bitvavo.php

```diff
@@ -126,9 +126,55 @@ function createCurl($url, $method, $params) {
       curl_setopt($curl, CURLOPT_URL, $url);
     }
     curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+    curl_setopt($curl, CURLOPT_HEADER, true);
+
     return $curl;
   }
 
+  function processRatelimit($header) {
+
+    $this->ratelimit = [];
+
+    $header = explode("\n", $header);
+    $header = array_map(function($line) {return trim($line);}, $header);
+    foreach ($header as $h) {
+      if (preg_match('/^bitvavo-ratelimit-(limit|remaining|resetat): +(\d+)$/', $h, $match)) {
+        list(, $key, $value) = $match;
+        $this->ratelimit[$key] = (int) $value;
```

Comment:
This is very usefull.
One small bugfix for this PR on 32 bit systems.
Casting the resetat key to int will result in a max value of an int32: 2147483647
Casting to a float resolves this problem.

`$this->ratelimit[$key] = (float)$value;`